include/realtime_urdf_filter/renderable.h

```diff
@@ -33,9 +33,8 @@
 
 #include <GL3/gl3.h>
 #include <tf/tf.h>
-// this is necessary for diamondback. for more recent ROS versions, use:
-// #include <urdf_interface/color.h>
-#define ParseError std::runtime_error
```

Comment:
This was a fix introduced by @mehditlili in the past, not sure why he didn't include exception.h instead. The fix in this PR seems to be cleaner though, so I'd rather do it this way.